README.md

```diff
@@ -160,14 +160,14 @@ test('skip() method with message', t => {
 
 ### `only` tests
 
-If `node-core-test` is started with the `--test-only` command-line option, it is
+If `test` is started with the `--test-only` command-line option, it is
 possible to skip all top level tests except for a selected subset by passing
 the `only` option to the tests that should be run. When a test with the `only`
 option set is run, all subtests are also run. The test context's `runOnly()`
 method can be used to implement the same behavior at the subtest level.
 
 ```js
-// Assume node-core-test is run with the --test-only command-line option.
+// Assume test is run with the --test-only command-line option.
```

Comment:
I think we need ticks to remove the ambiguity around the package name.
```suggestion
// Assume `test` is run with the `--test-only` command-line option.
```

Reply:
Actually, `test` is a bash builtin too, so it's also not a good cli name 🤔

Reply:
not sure what to do about this. do you have an idea?

Reply:
We could keep `node-core-test`, only use `npx test` (and `yarn test` on Yarn projects), or possibly both.